cpp/src/prims/count_if_v.cuh

```diff
@@ -35,9 +35,9 @@ namespace detail {
 
 template <typename vertex_t, typename VertexValueInputIterator, typename VertexOp>
 struct count_if_call_v_op_t {
-  vertex_t local_vertex_partition_range_first{};
-  VertexValueInputIterator vertex_value_input_first{};
-  VertexOp v_op{};
+  vertex_t local_vertex_partition_range_first;
+  VertexValueInputIterator vertex_value_input_first;
```

Comment:
Can't we default initialize iterators as well? Or is this because of possible thrust::transform_iterator with lambda?

Reply:
My reasoning with removing all default initializers is that partially initializing the struct does not make any sense and it is only ever initialized in a single place, so it feels "cleaner" to initialize all or nothing

Nothing I feel strongly about though